server/src/com/thoughtworks/go/server/web/IgnoreResolver.java

```diff
@@ -76,6 +77,10 @@ private boolean isGoingToAbout(HttpServletRequest request) {
         return StringUtils.contains(request.getRequestURI(), request.getContextPath() + "/about");
     }
 
+    private boolean isGoingToPlugin(HttpServletRequest request) {
+        return StringUtils.contains(request.getRequestURI(), request.getContextPath() + "/plugins/interact");
```

Comment:
`/plugins/interact` comes from [here](https://github.com/gocd/gocd/pull/1116/files#diff-52263857e788ae1666b5b0b9c5b552d9R210). `{requestName}` maps to what request name for plugin, like [here](https://github.com/gocd/go-plugins/pull/14/files#diff-8285148299afe93bde971ae9ebb96a67R81)